eessi/testsuite/tests/apps/lammps/lammps.py

```diff
@@ -8,18 +8,23 @@
 
 from eessi.testsuite import hooks, utils
 from eessi.testsuite.constants import *  # noqa
+from eessi.testsuite.eessi_mixin import EESSI_Mixin
 
 
-class EESSI_LAMMPS_base(rfm.RunOnlyRegressionTest):
-    scale = parameter(SCALES.keys())
+class EESSI_LAMMPS_base(rfm.RunOnlyRegressionTest, EESSI_Mixin):
     valid_prog_environs = ['default']
     valid_systems = ['*']
     time_limit = '30m'
+
```

Comment:
we can use an easy workaround to move those 3 variables into the mixin class, for example for time_limit:

```py
    @run_after('init')
    def set_time_limit(self):
        if not self.time_limit:
            self.time_limit = '1h'
```

Reply:
Great, thanks Sam for this clever solution!